packages/amplication-client/src/Components/CommitSelectorItem.tsx

```diff
@@ -17,12 +19,17 @@ export const CommitSelectorItem = ({ commit }: Props) => {
     <label className={`commit-selector__hour`}>{createdAtHour}</label>
   );
 
+  const truncateCommitMessage = truncateWithEllipsis(
+    commit?.message.trim(),
+    15,
+    "No commit message"
+  );
+
```

Comment:
you should use useMemo for things like this otherwise the function will be called with every render